backend/graphql/types/userType.ts

```diff
@@ -37,6 +37,7 @@ const userType = gql`
     role: Role!
     phoneNumber: String
     branchId: ID!
+    title: String
```

Comment:
Let's make this non-nullable as well
```suggestion
    title: String!
```